packages/validate/src/ValidateMixin.js

```diff
@@ -15,7 +15,7 @@ const pascalCase = str => str.charAt(0).toUpperCase() + str.slice(1);
 export const ValidateMixin = dedupeMixin(
   superclass =>
     // eslint-disable-next-line no-unused-vars, no-shadow, max-len
-    class ValidateMixin extends CssClassMixin(ObserverMixin(LocalizeMixin(SlotMixin(superclass)))) {
+    class ValidateMixin extends ObserverMixin(LocalizeMixin(SlotMixin(superclass))) {
       /* * * * * * * * * *
```

Comment:
removes the CssClassMixin as it "breaks" the prototype chain...

should use reflect: true for most properties and should for backward compatibility still add the css classes

=> separate MR